### PR TITLE
Add armor as a parameter

### PR DESCRIPTION
**Pull request by TheoGoudout**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -29,6 +29,11 @@ func main() {
 			Usage:  "passphrase for the key",
 			EnvVar: "PLUGIN_PASSPHRASE,GPGSIGN_PASSPHRASE,GPG_PASSPHRASE",
 		},
+		cli.BoolFlag{
```
Comment (tboerger) [inline main.go]: As this is a breaking change to the current behavior I would prefer a `Value: true,` attribute for the flag to enable it by default.

**Review by tboerger — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (TheoGoudout) [reply main.go]: There is no ``Value`` attribute for ``BoolFlag``. I'm going to change it for a "no-armor" Bool flag instead.

Comment (tboerger) [reply main.go]: ~~Ah, it's named `Default`, not `Value`... Sorry!~~

That's what I'm using for other projects, so there must be that flag:

```
			&cli.BoolFlag{
				Name:    "log-color",
				Value:   true,
				Usage:   "enable colored logging",
				EnvVars: []string{"OVERVIEW_LOG_COLOR"},
			},
```

Comment (tboerger) [reply main.go]: Ok, we are still using urfave/cli v1 here, so it works totally different.

```patch
-		cli.BoolFlag{
+		cli.BoolTFlag{
```

Comment (TheoGoudout) [reply main.go]: Thanks, that's what I was looking for

**Review by tboerger — APPROVED** [review]

**Pull request merged**